core/src/main/scala/za/co/absa/fadb/DBFunction.scala

```diff
@@ -74,7 +74,7 @@ abstract class DBFunction[I, R, E <: DBEngine[F], F[_]](functionNameOverride: Op
 
 /**
  *  `DBFunctionWithStatus` is an abstract class that represents a database function with a status.
- *  It extends the [[DBFunction]] class and adds handling for the status of the function invocation.
+ *  It extends the `DBFunction` class and adds handling for the status of the function invocation.
```

Comment:
What's the rule behind this? I can see references to abstract class (in this example) with both, `[[` and ```, across the code

Reply:
The square brackets are better since when you change class name it updates automatically, that's not the case with backtics. I have replaced the backtics for brackets everywhere for object names.